src/app/medical_record/repositories/mysql.go

```diff
@@ -16,6 +17,17 @@ type repository struct {
 	DB *gorm.DB
 }
 
+// DeleteByID implements medicalrecord.Repositories
+func (repo *repository) DeleteByID(id string) error {
+	result := repo.DB.Where("ID = ?", id).Delete(new(MedicalRecord))
+
+	if result.RowsAffected == 0 {

```

Comment:
please add condition if error == nil to avoid the error always return `not found`

Reply:
Applied https://github.com/ALTERA-CAPSTON-41/backend/pull/68/commits/b6b73cbe2c1295b7df6e63c66342dddd3ef17189.